src/MongoDB.Driver/Linq/Processors/Pipeline/MethodCallBinders/JoinSerializer.cs

```diff
@@ -148,6 +148,16 @@ public bool IsAtEndOfFile()
                 return _parent.IsAtEndOfFile();
             }
 
+            public void PopSettings()
+            {
+                _parent.PopSettings();
```

Comment:
@DmitryLukyanov you had `throw new NotImplementedException` but this is better right?

Reply:
Agreed.